tldw_Server_API/tests/Audit/test_audit_export_endpoint.py

```
@@ -62,7 +62,7 @@ async def _get_client(monkeypatch):
     from tldw_Server_API.app.main import app
 
     transport = httpx.ASGITransport(app=app)
-    async with httpx.AsyncClient(transport=transport, base_url="http://test", timeout=None) as client:
+    async with httpx.AsyncClient(transport=transport, base_url="http://test", timeout=None) as client:  # nosec B113
```

Ruff (0.15.1)

[error] Probable use of `httpx` call with timeout set to `None`

(S113)